src/common/browser-adapters/chrome-adapter.ts

```diff
@@ -1,5 +1,6 @@
 // Copyright (c) Microsoft Corporation. All rights reserved.
 // Licensed under the MIT License.
+import { browser } from 'webextension-polyfill-ts';
```

Comment:
have you validated this change on edge? as per their github page, they don't support edge. - https://github.com/mozilla/webextension-polyfill#supported-browsers

Reply:
they havn't validated after migrating to chrome based edge

Reply:
I think they refer to old edge but I will properly validate and come back with what I find.

Reply:
Tested on **Microsoft Edge Version 79.0.279.0 (Official build) dev (64-bit)** and it works as expected!